src/game/engine/player.py

```diff
@@ -64,8 +70,31 @@ def draw(self, screen, offset):
         screen: Any
              The screen to draw the player onto
         """
-        draw_position=self.position.move(offset[0],offset[1])
-        screen.blit(self.image, draw_position)
+        screen.blit(self.image,self.position)
+    
+    def updateJump(self):
+        x, y, width, height = self.position
```

Comment:
self.position, sadly can't be deconstructed into those values.
you have to use `pygame.Rect`'s ways of getting them (I believe just)
```python
self.position.width
self.position.height
# etc refer to docs
```